gnucash/report/reports/standard/advanced-portfolio.scm

```diff
@@ -112,16 +100,9 @@ by preventing negative stock balances.<br/>")
      (gnc:make-multichoice-option
       gnc:pagename-general optname-brokerage-fees
       "g" (N_ "How to report commissions and other brokerage fees.") 'include-in-basis
-      (list (vector 'include-in-basis
-                    (N_ "Include in basis")
-                    (N_ "Include brokerage fees in the basis for the asset."))
-            (vector 'include-in-gain
-                    (N_ "Include in gain")
-                    (N_  "Include brokerage fees in the gain and loss but not in the basis."))
-            (vector 'ignore-brokerage
-                    (N_ "Ignore")
-                    (N_ "Ignore brokerage fees entirely."))
-            )))
+      (list (vector 'include-in-basis (N_ "Include brokerage fees in the basis for the asset"))
+            (vector 'include-in-gain (N_ "Include brokerage fees in the gain and loss but not in the basis"))
+            (vector 'ignore-brokerage (N_ "Ignore brokerage fees entirely")))))
```

Comment:
Also here I think the original value names were sufficient, though possible suggestions would be
(second value) "Include in gain/loss"
(last value) "Omit from report"